docs/installation.rst

```diff
@@ -0,0 +1,118 @@
+.. _installing_ansible_navigator:
+
+Installing `ansible-navigator` with execution environment support
+#################################################################
+
+.. contents::
```

Comment:
FWIW you could've kept MD: https://myst-parser.readthedocs.io/en/latest/sphinx/intro.html#extend-markdown-with-a-directive. MyST allows you to have the same as RST.

Reply:
yeah, wanted to give my rusty rst skills a run

Reply:
Right. FTR while I personally prefer RST, it seems to be more contributor-friendly to have (most of the?) content in MD. This is why I've integrated MyST primarily. Plus many projects in the ecosystem have done this over the past few years.